scripts/india_wris/util/groundWater.json

```diff
@@ -0,0 +1,122 @@
+[
```

Comment:
Can you add documentation for the json format / properties in the README

Reply:
Added documentation for json files under respective README files.